metabase/metabase
# Extract right click popover from NativeQueryEditor

One more step to solve #18536

Here we simplify `NativeQueryEditor` a bit more.

This should not only make the feature easier to understand, but also bring to light how to excise global Ace so we can replace it with react-ace.

## How to Test

1. Ask a question
2. Native query
3. In the editor, type some query like `select * from PRODUCTS;`
4. Select, then right-click the text you typed.

A Popover should be displayed with two buttons, fully functional as before this component extraction.

### Extracted Component
![image](https://user-images.githubusercontent.com/380816/138372433-8a0c984a-2e61-48eb-9b80-571ce1dd7b70.png)
